$NetBSD: patch-tools_build_src_tools_clang-darwin.jam,v 1.1 2023/05/23 18:23:43 tnn Exp $

Work around clang -x c -std=c++14 passed by pkgsrc wrappers
not working, by overriding w/ -std=c99 later in the cmd line.

--- tools/build/src/tools/clang-darwin.jam.orig	2023-04-10 13:47:41.000000000 +0000
+++ tools/build/src/tools/clang-darwin.jam
@@ -116,7 +116,7 @@ rule compile.c++ ( targets * : sources *
 
 actions compile.c bind PCH_FILE
 {
-    "$(CONFIG_COMMAND)" -x c $(OPTIONS) $(USER_OPTIONS) -D$(DEFINES) -I"$(INCLUDES)" $(.include-pch)"$(PCH_FILE)" -c -o "$(<)" "$(>)"
+    "$(CONFIG_COMMAND)" -x c $(OPTIONS) $(USER_OPTIONS) -std=c99 -D$(DEFINES) -I"$(INCLUDES)" $(.include-pch)"$(PCH_FILE)" -c -o "$(<)" "$(>)"
 }
 
 actions compile.c++ bind PCH_FILE
